### DISCUSSION TITLE
ACP-41: Remove Pending Stakers

### DISCUSSION THREAD

**Discussion by avalanche-foundation-admin** [avalanche-foundation/ACPs#42] [Discussions]
Remove user-specified `StartTime` for stakers. Start the staking period for a staker as soon as their staking transaction is accepted. This greatly reduces the computational load on the P-chain, increasing the efficiency of all Avalanche Network validators.

https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/41-remove-pending-stakers/README.md

**Comment by aphexmunky** [comment]
Is there any more information around the previous thinking for implementing it like this and what has changed around that? My assumption was that you don't want to conflate the counting of the limits with the preference of the chain. Generally in-favour of the change otherwise, I see no other useful reason to have a pending period

**Reply by dhrubabasu** [reply]
The original staking mechanism required explicit start and end times. However, with the P-chain timestamp changes from the Banff upgrade, the explicit start time is no longer needed and only impedes UX
